src/fhiry/parallel.py

```diff
@@ -1,4 +1,4 @@
-from src.fhiry import Fhiry, Fhirndjson
+from fhiry import Fhiry, Fhirndjson
```

Comment:
Thank you @Mandalka for pointing this out. I think we should use from . import Fhiry, Fhirndjson for relative imports. The tests should pass after that. Thanks!